jusleg/soen341
# bower.json and prototype of conversation between person completed

To test: 

Go to terminal and do ==> node index
2) then open two different tab (representing two people)
3) chat between the two tabs.
4) observe the live chat room
